.dotnet/src/Custom/VectorStores/VectorStoreClient.cs

```diff
@@ -551,5 +566,77 @@ public virtual CreateBatchFileJobOperation CreateBatchFileJob(
         RequestOptions options = cancellationToken.ToRequestOptions();
 
         return CreateBatchFileJob(vectorStoreId, content, waitUntilCompleted, options);
+    }
+
+    internal virtual PipelineMessage CreateCreateVectorStoreFileBatchRequest(string vectorStoreId, BinaryContent content, RequestOptions options)

```

Comment:
Also a nit, but Jose/Travis might prefer if the request creation helpers lived in the .Protocol.cs file with the protocol methods -- IIRC that is the convention they've been following.

Reply:
happy to change if I should

Reply:
Yes, please! As mentioned in an earlier comment, it's really just about keeping the "boring" parts of protocol implementation out of the smaller files focused on more "interesting" convenience things.

Reply:
fixed